Header header
time ros_stamp
time camera_stamp
